### ISSUE TITLE
Create TFA OSS Usage Analyzer

### ISSUE THREAD

**Issue by seanpmorgan** [tensorflow/addons#236]
We've been using the TF contrib analyzer provided by @kingspp in order to triage components of tf.contrib to be moved to Addons: 
https://tf-contrib-analyzer.herokuapp.com/
https://github.com/kingspp/tf-contrib-analyzer

Per our charter... we'll need to analyze the usage of Addons as part of our periodic review process. As part of a fork we may want to add some changes to improve the accuracy and push them upstream. The first one that comes to mind is to ignore `.rst` and `.md` files which are more than likely false positives.

cc @dynamicwebpaige as this was discussed in the Monthly meeting and there may be interest in setting this up for all of TensorFlow if applicable.

**Comment by kingspp** [comment]
Let me know if you guys need any help.

**Comment by WindQAQ** [comment]
@seanpmorgan Hi Sean, is this still active?

**Comment by seanpmorgan** [comment]
Hey Tzu-Wei. Yeah... still active but not a priority. I don't think we'll be gauging OSS adoption until ~6months after TF2 release.

**Comment by WindQAQ** [comment]
Sure,  thanks for the clarification  :smiley:

**Comment by bhack** [comment]
@seanpmorgan  It is not the contrib project resurrection but I have a quick script to make a rought count of tensorflow_addons repos on github:
https://gist.github.com/bhack/885b79e57a6c4a59f1b1e5e12df74884

We have `used by` Github counter that check only `requirements.txt` that is at 191 now. 
The script code search instead count a `Total Repos: 363` (without filters like addons repo itself etc..)

**Comment by bhack** [comment]
I've another stub for an histogram sampling in the first 100 repositories:
 https://gist.github.com/bhack/f2a2e81e556545ce888b1b7631e9e06d .

![plot](https://user-images.githubusercontent.com/1710528/78034230-bd1f6b00-7367-11ea-8537-5f26929b8cb0.png)

**Comment by seanpmorgan** [comment]
> I've another stub for an histogram sampling in the first 100 repositories:
> https://gist.github.com/bhack/5fb3d721a995a672a236f1eede016340
> 
> ![plot](https://user-images.githubusercontent.com/1710528/78034230-bd1f6b00-7367-11ea-8537-5f26929b8cb0.png)

Awesome thanks @bhack this is very useful. We're also interested in what addons have no usage as well to help us determine if there are bugs/implementation problems, duplicated code on different repos, etc.

Thanks a bit trickier to do but certainly useful

**Comment by bhack** [comment]
> We're also interested in what addons have no usage as well to help us determine if there are bugs/implementation problems, duplicated code on different repos, etc.

What do you mean?

**Comment by seanpmorgan** [comment]
As in which specific addons modules are / are not being used.

**Comment by bhack** [comment]
Yes no usage it is quite hard to define for pattern matching. For this subtopic I am mainly interested in the Google brain/TF perimeters as you know we could to achieve an important policy enforcing goal. It is hard in these days to find refence impl of non Google papers in Tensorflow so at least  having Google Research/Brain/TF team itself contributing in addon could help on the points that you have exposed.
See edge cases of non use like this  https://github.com/tensorflow/io/issues/839#issuecomment-607362644 (a Google paper, no official impl, many Pytorch impl, not finalizend in addons, implemented in TF by Mozilla on third party repo etc..)

**Comment by seanpmorgan** [comment]
TensorFlow Addons is transitioning to a minimal maintenance and release mode. New features will not be added to this repository. For more information, please see our public messaging on this decision:
[TensorFlow Addons Wind Down](https://github.com/tensorflow/addons/issues/2807)

Please consider sending feature requests / contributions to other repositories in the TF community with a similar charters to TFA:
[Keras](https://github.com/keras-team/keras)
[Keras-CV](https://github.com/keras-team/keras-cv)
[Keras-NLP](https://github.com/keras-team/keras-nlp)
